### DISCUSSION TITLE
How to helperText show html

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#7012] [Help] [answered]
I want make breakline and bold in helperText.
![image](https://github.com/filamentphp/filament/assets/10229660/cd55d7b9-8398-422a-af64-9757c2f8bf0b)
![image](https://github.com/filamentphp/filament/assets/10229660/f71a728d-e6e8-4a1d-bf31-5b43578375c2)

**Comment by danharrin** [comment, marked as answer]
`helperText(new HtmlString('html here'))`

**Reply by AFSDARIF** [reply]
Thank You
